force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -241,8 +338,11 @@ public with sharing class FlowControlLoopsCollections {
         Set<String> stringSet = new Set<String>();
 
         //add strings to the set
-
-        return null; // Replace null with the variable you used to store the result
+        for (String str : inputList) {
+            stringSet.add(str);
+        }
```

Comment:
A loop is a reasonable way to add items to a set from a list -- but can you find a Set method that will do it all in one go (one line of code)?